portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,29 @@ function addRandomFact() {
   const factContainer = document.getElementById('fact-container');
   factContainer.innerText = randomFact;
 }
+
+function getFromServlet(){
+    console.log("Fetching from sevlet");
+
+    const fetchPromise = fetch('/data');
+
+    fetchPromise.then(handle);
+}
+
+function handle(response){
+    console.log("Handling promise");
+
+    const responseText = response.text();
+
+    responseText.then(addToSite);
+}
+
+function addToSite(saying){
+    console.log("Adding " + saying + " to site");
+
+    const sayingContainer = document.getElementById('saying-container');
+    
+    sayingContainer.innerHTML = saying;
+    }
```

Comment:
fix indenting here

Reply:
In what way? Right now, they're indented under the function. Should I indent them further right or left?

Reply:
Ah yea, the 4 space indentation threw me off.  It looks like the only issue is the closing brace on line 52

Reply:
oh, gotcha. I thought you meant the interior of the function, not the brace.